### DISCUSSION TITLE
Accountability of signing quorum from BDK wallet

### DISCUSSION THREAD

**Discussion by jurvis** [bitcoindevkit/bdk#1088] [Q&A]
As a wallet user, I would like to figure out which signers were used to add their signatures to the output script, is that something possible to do with pre-existing BDK APIs?

**Comment by LLFourn** [comment]
Hey @jurvis -- I think this is an interesting feature. it is certainly theoretically possible to do for the general problem of "which branches of the minscript policy were satisfied by this witness". Not sure if we can easily ask this question with the miniscript API as it is today.

cc @sanket1729
